openverse-api/catalog/api/models/media.py

```diff
@@ -131,6 +108,8 @@ class AbstractMediaReport(models.Model):
     deleted content.
     """
 
+    BASE_URL = 'https://search.creativecommons.org/'
```

Comment:
Should we update the BASE_URL now?

Reply:
LGTM and do it in a separate PR to rename the base url.